src/Mailer/Email.php

```diff
@@ -241,6 +241,14 @@ class Email implements JsonSerializable, Serializable
      */
     public $headerCharset;
 
+    /**
+     * The email transfer encoding used.
+     * If null, the $charset property is used for determined the transfer encoding.
+     *
+     * @var string|null
+     */
+    public $transferEncoding;
```

Comment:
Does it make sense to add another public properties and a pair of getter/setter for that?

Reply:
I think no, but i follow what was done. 
You can see the charset or headerCharset properties.

Reply:
Those are public for BC reasons I suppose. I think we can make the new property protected.

Reply:
Yeah the property should be protected.

Reply:
Ok i fix it